test/elpy-yapf-fix-code-test.el

```diff
@@ -15,3 +15,28 @@
                       (buffer-be
                        "x = 1"
                        "y = 2_|_"))))))
+
+(ert-deftest elpy-yapf-fix-code-in-region-should-retain-line-and-column ()
+  (let* ((pyversion (getenv "TRAVIS_PYTHON_VERSION"))
+         (yapf-not-supported (or (string< pyversion "2.7")
+                                 (and (not (string< pyversion "3.0"))
+                                      (string< pyversion "3.4")))))
+    (unless yapf-not-supported
+      (elpy-testcase ()
+                     (set-buffer-string-with-point
+                      "x  =  1"
+                      "_|_y     = 2"
+                      "z        = 3"
+                      )
+                     (elpy/mark-region (point) (point-max))
+
+                     (elpy-yapf-fix-code)
+
+                     (should (not mark-active))
+                     (should
+                      (buffer-be
+                       "x  =  1"
+                       "y = 2"
+                       "z = 3"
+                       "_|_"
```

Comment:
`      (buffer-contents-differ "x  =  1\ny = 2\nz = 3\n_|_\n" "x  =  1\ny = 2\nz = 3\n_|_")))
`
 Any idea why `\n` is getting added in the end again? @jorgenschaefer